rare/components/tabs/games/game_list.py

```diff
@@ -86,8 +87,10 @@ def init_ui(self, icon_view=True):
                 self.uninstalled_names.append(game)
 
         # add uninstalled games
-        for game in self.uninstalled_names:
```

Comment:
Something is missing as there it crashes here with
```
AttributeError: 'GameList' object has no attribute 'uninstalled_games'
```

Reply:
I fixed it